enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/HighLimit.java

```diff
@@ -54,7 +54,7 @@
 
     public HighLimit()
     {
-        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
+        super( STORE_VERSION, 8, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
```

Comment:
Should this not be a separate store version? I also think there should be a `RecordFormats$Factory` implementation for the 3.0 format, with a `META-INF/services` file, so we can load the old implementation.